example/example2.c

```diff
@@ -139,41 +139,106 @@ static BOOL RegisterBasicAUMID(LPCWSTR aumid, LPCWSTR displayName, LPCWSTR iconU
     return TRUE;
 }
 
-int main(int argc, char ** argv)
+// Déclarer l'événement global
+HANDLE g_hEvent = NULL;
+
+// Callback functions
+void toastActivated(void *userData) {
+    printf("Toast activated without action.\n");
+    // Signaler que le callback a été appelé
+    if (g_hEvent != NULL) {
+        SetEvent(g_hEvent);
+    }
+}
+
+void toastActivatedAction(void *userData, int actionIndex) {
+    if (actionIndex == 0) {
+        printf("Button 1 clicked.\n");
+    } else if (actionIndex == 1) {
+        printf("Button 2 clicked.\n");
+    } else {
+        printf("Unknown action clicked.\n");
+    }
+    // Signaler que le callback a été appelé
+    if (g_hEvent != NULL) {
+        SetEvent(g_hEvent);
+    }
+}
+
+void toastDismissed(void *userData, WTLC_DismissalReason state) {
+    switch (state) {
+        case WTLC_DismissalReason_UserCanceled:
+            printf("Toast dismissed by user.\n");
+            break;
+        case WTLC_DismissalReason_ApplicationHidden:
+            printf("Toast dismissed by application.\n");
+            break;
+        case WTLC_DismissalReason_TimedOut:
+            printf("Toast timed out.\n");
+            break;
+        default:
+            printf("Toast dismissed with unknown reason.\n");
+            break;
+    }
+    // Signaler que le callback a été appelé
+    if (g_hEvent != NULL) {
+        SetEvent(g_hEvent);
+    }
+}
+
+void toastFailed(void *userData) {
+    printf("Toast failed to display.\n");
+    // Signaler que le callback a été appelé
+    if (g_hEvent != NULL) {
+        SetEvent(g_hEvent);
+    }
+}
+
+INT WINAPI wWinMain()
```

Comment:
It is not portable declaraion. For example, MSVC 2019 x86 build fails with error:
```
C:\WINDOWS\system32\cmd.exe /C "cd . && "C:\Program Files\CMake\bin\cmake.exe" -E vs_link_exe --intdir=CMakeFiles\example2.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MIB055~1\2022\COMMUN~1\VC\Tools\MSVC\1439~1.335\bin\Hostx64\x86\link.exe /nologo CMakeFiles\example2.dir\example\example2.c.obj  /out:example2.exe /implib:example2.lib /pdb:example2.pdb /version:0.0 /machine:X86 /INCREMENTAL:NO /subsystem:windows  wintoastlibc_static.lib  shlwapi.lib  ole32.lib  user32.lib  advapi32.lib  ole32.lib  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib && cd ."
LINK: command "C:\PROGRA~1\MIB055~1\2022\COMMUN~1\VC\Tools\MSVC\1439~1.335\bin\Hostx64\x86\link.exe /nologo CMakeFiles\example2.dir\example\example2.c.obj /out:example2.exe /implib:example2.lib /pdb:example2.pdb /version:0.0 /machine:X86 /INCREMENTAL:NO /subsystem:windows wintoastlibc_static.lib shlwapi.lib ole32.lib user32.lib advapi32.lib ole32.lib kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST:EMBED,ID=1" failed (exit code 1120) with the following output:
MSVCRT.lib(exe_winmain.obj) : error LNK2019: unresolved external symbol _WinMain@16 referenced in function "int __cdecl __scrt_common_main_seh(void)" (?__scrt_common_main_seh@@YAHXZ)
example2.exe : fatal error LNK1120: 1 unresolved externals
ninja: build stopped: subcommand failed.
03:12:30: The process "C:\Program Files\CMake\bin\cmake.exe" exited with code 1.
```